### PR TITLE
9300 allow three list cards per row

### PR DESCRIPTION
**Pull request by tbrlpld**
# Description

Adds the options to show 3 cards per row to the listing block. Because this block is used in other places, we could not just change the default. The requirement for 3 per row is coming from the MozFest homepage.

Link to sample test page: http://mozfest.localhost:8000/en/
Related PRs/issues: https://torchbox.monday.com/boards/1334760528/pulses/1336149300

# Checklist

**Tests**
- [-] Is the code I'm adding covered by tests?

**Changes in Models:**
- [-] Did I update or add new fake data?
- [x] Did I squash my migration?
- [x] Are my changes [backward-compatible](https://github.com/mozilla/foundation.mozilla.org/blob/main/docs/workflow.md#django-migrations-what-to-do-when-working-on-backward-incompatible-migrations). If not, did I schedule a deploy with the rest of the team?

**Documentation:**
- [-] Is my code documented?
- [-] Did I update the READMEs or wagtail documentation?

**Merge Method**
**💡❗Remember to use squash merge when merging non-feature branches into `main`**

### CODE REVIEW

File: network-api/networkapi/wagtailpages/templates/wagtailpages/blocks/listing_block.html [hunk 1]
```diff
@@ -7,10 +7,21 @@
...
+    <ul class="
+        tw-grid
+        tw-mb-24
+        tw-pl-0
+        tw-list-none
+        tw-gap-8
+        medium:tw-gap-24
+        medium:tw-grid-cols-2
+        {% if value.cards_per_row == "3" %}
+            large:tw-grid-cols-3
```
Comment (tbrlpld) [inline network-api/networkapi/wagtailpages/templates/wagtailpages/blocks/listing_block.html:19]: I tried at `medium` before, but that does not look right. It's not enough space.

File: network-api/networkapi/wagtailpages/templates/wagtailpages/blocks/listing_block.html [hunk 2]
```diff
@@ -22,7 +33,16 @@ <h2 class="tw-h3-heading tw-mb-0 tw-pb-8 medium:tw-pb-16">{{ self.heading }}</h2
...
+                        <picture>
+                            {% image card.image fill-350x197 as img_small %}
+                            {% image card.image fill-700x394 as img_small_2x %}
+                            {% image card.image fill-510x287 as img_large %}
+                            {% image card.image fill-1020x574 as img_large_2x %}
+                            {# Larger screens use smaller image because multiple images are displayed per row, making each image smaller. #}
+                            <source media="(min-width: 768px)" srcset="{{ img_small.url }}, {{ img_small_2x.url }} 2x">
+                            <source srcset="{{ img_large.url }}, {{ img_large_2x.url }} 2x">
+                            <img src="{{ img_large.url }}" alt="{{ card.alt_text }}" width="{{ img_large.width }}" height="{{ img_large.height }}">
+                        </picture>
```
Comment (tbrlpld) [inline network-api/networkapi/wagtailpages/templates/wagtailpages/blocks/listing_block.html:45]: I noticed that the image look washed out on small screens. This should fix it and use higher resolution on small screens as the image is effectively bigger.

**Comment by chris-lawton** [comment]
Thanks @tbrlpld - FE looks good.

**Review by kevinhowbrook — APPROVED** [review]
Thanks @tbrlpld , good looking out for the other sections :+1: 

I'll fix the pipeline and merge this in

**Pull request merged**